zipkin-lens/src/zipkin/trace.js

```diff
@@ -266,6 +266,14 @@ function addLayoutDetails(
   }
 }
 
+export function rootServiceAndSpanName(root) {
+  const { span } = root.queueRootMostSpans().shift();
```

Comment:
I think maybe we should be less precise on a headless span, and not name the detail pane.

so, use `root.span()` and if undefined, don't name this trace.